client/src/pages/NrMap.js

```diff
@@ -33,62 +33,69 @@ const Cards = styled.div`
   margin: 10px;
 `;
 
-class NrMap extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      latitude: 52.520008,
-      longitude: 13.404954,
-      rec: [],
-      plcDtl: [],
-    };
-    this.googleMapRef = React.createRef();
-  }
-  componentDidMount() {
+const NrMap = () => {
+  const [coordinates, setCoordinates] = useState({
+    latitude: 52.520008,
+    longitude: 13.404954,
```

Comment:
where do these defaults come from?  Might be better to declare them as constants and put them at the top of the file or something like that, also easier to find back later when searching for something instead of using "magic" numbers or string values.

Reply:
I agree. moved to the top of the file as global variables